charts/consul/values.yaml

```diff
@@ -1859,6 +1859,76 @@ connectInject:
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
 
+  # Configures consul-cni plugin for Consul Service mesh services
+  cni:
+    # If true, then all traffic redirection setup with use the consul-cni plugin
+    # @type: boolean
+    enabled: false
+ 
+    # If multus plugin is enabled or not. If true, the CNI installer will generate a consul CNI configuration
+    # in a location for the multus program to use. Also, the helm chart will create a custom resource for
+    # NetworkAttachmentDefintion that multis will consume. The values for cniBinDir and cniNetDir must also be 
+    # changed for multus to work.
```

Comment:
Could you describe how they need to be changed?

Reply:
I added an example for GKE to the comment.